contracts/membershipCheckers/DAOStackMembershipChecker.sol

```diff
@@ -0,0 +1,28 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./IMembershipChecker.sol";
+import "../daoStandards/IDAOStack.sol";
+
+/// @title NewMembershipChecker
```

Comment:
put relevant title, notice etc, please

Reply:
Done ;)

Reply:
Thanks! We're talking about anything Open source and Bounties related on our discord server. Join us there :) https://discord.gg/dRmh62Yj